pynest/nest/tests/test_refractory.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+#
+# test_refractory.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+
+import numpy as np
+
+import nest
+
+"""
+Assert that all neuronal models implement refractory period correctly.
+
+Details
+-------
+Submit the neuron to a constant excitatory current so that it spikes in the
+[0, 50] ms.
+A ``spike_detector`` is used to detect the time at which the neuron spikes and
+a ``voltmeter`` is then used to make sure the voltage is clamped to ``V_reset``
+during exactly ``t_ref``.
+"""
+
+
+# --------------------------------------------------------------------------- #
+#  Models, specific parameters
+# -------------------------
+#
+
+ignore_model = [
+    "aeif_cond_alpha_RK5",
+    "aeif_cond_alpha_multisynapse",
+    "aeif_cond_beta_multisynapse",
+    "amat2_psc_exp",
+    "ginzburg_neuron",
+    "hh_cond_exp_traub",
```

Comment:
Why do you need to ignore this model? Isn't that the one you are fixing in this PR?